vllm/model_executor/models/jamba.py

```diff
@@ -420,6 +420,9 @@ def __init__(self, *, vllm_config: VllmConfig, prefix: str = ""):
 
         self.make_empty_intermediate_tensors = (
             self.model.make_empty_intermediate_tensors)
+        self.max_batch_size = (vllm_config.pad_for_cudagraph(
+            self.scheduler_config.max_num_seqs)
+                               if self.scheduler_config else 8192 + 2)
```

Comment:
@tlrmchlsmth do you know when `self.scheduler_config` will be `None` here?

Reply:
No, I don't. @mzusman do you know?

Reply:
Shouldn't be `None`, just added it as a safety guard -  It was added before the whole `vllm_config` was available inside the modeling file

Reply:
thanks! can you try to remove it then?

Reply:
yeah, will open a PR shortly